axelrod/tests/unit/test_strategy_transformers.py

```diff
@@ -15,6 +15,20 @@ class TestClass(object):
 
 class TestTransformers(unittest.TestCase):
```

Comment:
Could we also add specific unit tests for those two extra classes you wrote in `strategy_transformers`: `DecoratorReBuilder` and `StrategyReBuilder`?